frontend/app/components/app-bar.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTranslation } from 'react-i18next';
+
+import { LanguageSwitcher } from './language-switcher';
+import { Menu } from './menu';
+import { UserButton } from './user-button';
```

Comment:
For consistency with the rest of the application, these should be aliased imports (`~/`).